src/SleepLog.js

```diff
@@ -1,22 +1,15 @@
 class SleepLog {
 
   constructor(sleepData, userID) {
-    this.log = sleepData.filter((data) => data.userID === userID)
-      .map(userLog => {
```

Comment:
I removed this .map that we discussed

Reply:
Yeah, I thought I had pushed up that change but I guess it didn't go through :/